bases/rpcnodes/service.yaml

```diff
@@ -11,20 +11,30 @@ spec:
 apiVersion: cloud.google.com/v1
 kind: BackendConfig
 metadata:
-  name: stage-api-backendconfig
+  name: api-backendconfig
 spec:
   healthCheck:
     port: 1317
     requestPath: /node_info
     type: HTTP
 ---
+apiVersion: cloud.google.com/v1
+kind: BackendConfig
+metadata:
+  name: rpc-backendconfig
+spec:
+  healthCheck:
+    port: 26657
+    requestPath: /healthz
```

Comment:
I tried this `localhost:26656/healthz` on a3p but got a 404. What service is exposing /healthz ?

Reply:
Should be http://localhost:26657/healthz

Reply:
Actually, should be http://localhost:26657/abci_info